src/components/TabMenu/style.css

```diff
@@ -0,0 +1,37 @@
+.TabMenu {
+  display: flex;
+  align-items: flex-end;
+  padding: 0;
+  margin: 0;
+  list-style-type: none;
+}
+
+.leftAlign {
+  justify-content: flex-start;
+}
+
+.centerAlign {
+  justify-content: center;
+}
+
+.rightAlign {
+  justify-content: flex-end;
+}
+
+/* SimpleTabMenu uses these */
+.itemPadding {
+  padding: 16px 0;
+
+  /* reset default button styles */
+  background-color: transparent;
+  border: none;
+  color: inherit;
+  text-decoration: none;
+  display: block;
+  border-radius: 0;
+}
+
+.xsTextSize { font-size: 14px; }
```

Comment:
Are there no font-sizes as css custom properties in Rover yet? Either way, this should be configurable rather than static

Reply:
Good catch, i put these as placeholder because we didn't have css variables for font sizes other than small yet... but this is a good reminder to standardize on what those sizes should be